ompi/mca/fcoll/static/fcoll_static_file_read_all.c

```diff
@@ -646,8 +639,8 @@ mca_fcoll_static_file_read_all (mca_io_ompio_file_t *fh,
                                                                 global_iov_count,
                                                                 sorted);
                         if (current_index[i] == -1){
-                            bytes_per_process = 0; /* no more entries left
-                                                      to service this request*/
+                            bytes_per_process[i] = 0; /* no more entries left
```

Comment:
the way I see it, this code sequence is in master the same way ( i.e. bytes_per_process[i]) since March and it looks correct to me (bytes_per_process is an array). So go ahead with that commit.